chat-plugins/dice-commands.js

```diff
@@ -1,185 +1,164 @@
-/* Dice commands chat-plugin
- * by jd
-*/
-var tax = 0.10; // 10%
+/*Dice Game plugin by SilverTactic (Silveee)*/
 
-function handleWinnings (bet) {
-	return bet - Math.round(bet * tax);
+'use strict';
+
+const GC = Rooms('gamechamber');
```

Comment:
how about change this line to: const GC = (Rooms('gamechamber') ? Rooms('gamechamber') : Rooms('lobby'));`` or something just in case?